extensions/adobe/experience/decisioning/activity-detail.example.2.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please add the required field "xdm:id" in the example.

Reply:
thanks! I refactored the PR to add strategy-detail schema and add strategy id to the new schema.